examples/3d/clearcoat.rs

```diff
@@ -25,7 +25,9 @@ use bevy::{
     math::vec3,
     pbr::{CascadeShadowConfig, Cascades, CascadesVisibleEntities},
     prelude::*,
-    render::{primitives::CascadesFrusta, texture::ImageLoaderSettings},
+    render::{
+        mesh::primitives::MeshBuilder, primitives::CascadesFrusta, texture::ImageLoaderSettings,
+    },
```

Comment:
Perhaps the `MeshBuilder` should be in the prelude? If people are expected to use `build` often, importing `bevy::render::mesh::primitives::MeshBuilder` every time might be annoying. Usually `build` is called automatically through `Into`/`From` though.

Reply:
Sure, I will include it